src/basilisp/lang/runtime.py

```diff
@@ -1113,7 +1113,10 @@ def apply_kw(f, args):
 
 def count(coll) -> int:
     try:
-        return len(coll)
+        if coll is None:
+            return 0
```

Comment:
Sorry for the nit, but could you move this check outside of the `try` block?

Reply:
sorry my mistake, it should be fine now.